tests/install_upgrade_operators/node_component/conftest.py

```
@@ -227,12 +230,32 @@ def hyperconverged_resource_before_np(admin_client, hco_namespace, hyperconverge
     )
 
 
+@pytest.fixture()
```

Ruff (0.16.0)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)